src/context/identification.tsx

```diff
@@ -26,9 +28,12 @@ export const IdentificationContextProvider: FunctionComponent = ({
     defaultSelectedIdType
   );
 
+  const resetSelectedIdType = (): void =>
+    setSelectedIdType(defaultSelectedIdType);
+
```

Comment:
May want to wrap this in a `useCallback` so that whenever its ancestors updates, this function will have the same reference. This helps with ensuring stable references for `useEffect` dependencies